core/doc/doxygen/MicroServices_ModuleProperties.md

```diff
@@ -7,15 +7,22 @@ The following properties are always set by the C++ Micro Services library and ca
 the module author:
 
  * `module.id` - The unique id of the module (type `long`)
- * `module.name` - The human readable name of the module (type `std::string`)
```

Comment:
I believe I understand why you made this change. Still, I would like to see the module.name property added by the framework, such that for basic bundles no custom manifest.json file is required.

We should be able to use the `US_MODULE_INITIALIZATION` macro to inject a string field into the custom section we were talking about in the General Discussion document. The existing object format parsers in branch 25-module-lifecycle-enhancements can then extract that information. Because this requires some more coding, I suggest to keep your changes for the sake of merging them rather sooner than later. I will create a separate issue to investigate this after the merge.